doc/03-omnifest/01-directive.md

```diff
@@ -196,26 +196,66 @@ otk.meta.kiwi:
 
 ## `otk.customization`
 
+A `otk.customization` is distinct from a `otk.define` because:
+- End-users will supply them
+- ?
+
 Customizations are conditional blocks that receive separate input through
-`otk compile -Cname=data`, a customization is considered to be active when it
-is passed data. If a customization is passed multiple times then the `defined`
-block is repeated multiple times, once for each input.
+a `customization` file `otk compile -C customizations.yaml` that contains a
+map of customization settings in yaml format, e.g.
+```yaml
+hostname = "lala"
+
+[[user]]
```

Comment:
This looks like TOML, or is that also valid YAML?

Reply:
Thanks, yeah, I will fix this, I just blindly added a more complex example from our tests without converting.